src/transmission/email.ts

```diff
@@ -0,0 +1,62 @@
+/** based on document Gemeinsame Grundsätze Technik, Anlage 7
+ * (see /docs/documents.md for more info)
+ */
```

Comment:
It is still unclear whether the automatic response is evaluated. If so, then probably not by the library, since it cannot send and receive the e-mails itself.